portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,48 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<Comments> comments = new ArrayList<>();
+  private ArrayList<String> commentString = new ArrayList<>();
+
+  /**
+   * Converts an ArrayList instance into a JSON string using the Gson library.
+   */
+  private String convertToJsonUsingGson() {
```

Comment:
Dave, do you have an opinion on this?

I'm kind of leaning toward suggesting that commentString be a parameter of this function, rather than being silently passed via the instance variable.
If you do that, you could then also make the method "static", since it wouldn't depend on the servlet state.

That way if you want to have more than one List converted into Json, you could easily reuse this function to do it.